Cylenium.Tests/DriverTests.cs

```diff
@@ -29,5 +29,19 @@ public void Customize_wait_with_negative_timeout_uses_default()
 
             Assert.That(wait.Timeout.Seconds, Is.EqualTo(10));
         }
+
+        [Test]
+        [Category("Driver")]
+        public void Return_window_size()
+        {
+            var wait = cy.Wait(timeout: -5);
+            var size = cy.WindowSize();
+
+            cy.Visit("https://google.com");
+            wait.Until(_ => cy.Get("[name='q']"));
+            var window = cy.WindowSize();
+
+            Assert.AreEqual(size, window);
```

Comment:
You can simplify the test quite a bit by only focusing on what you _really_ need for it.

```c#
var expectedSize = new Size(height: 600, width: 800); // set the expected size to test against
var size = cy.Visit("https://google.com").WindowSize(); // get the actual size of the window
Assert.AreEqual(size, expectedSize);
```

Also, there is some redundancy with the `wait`. If you are not using the newly created `wait` in more than one place, then there's no need to instantiate a whole new instance of it.

```c#
cy.Wait(timeout: 5).Until();
```

Using a negative timeout will result in using the default timeout, which is `10 seconds`, so there's no need to specify that. Just use:

```c#
cy.Wait().Until();
```

Another piece is that `cy.Get()`, and some other commands, use a WebDriverWait themselves, so this line:

```c#
wait.Until(_ => cy.Get("[name='q']"));
```

is actually doubling your wait time because you have the default 10 seconds from `wait` and the default 10 seconds from `cy.Get()`